runners/pvc_autoresizer_test.go

```diff
@@ -432,6 +432,9 @@ func createPVC(ctx context.Context, ns, name, scName, threshold, inodesThreshold
 	Expect(err).NotTo(HaveOccurred())
 
 	pvc.Status.Phase = corev1.ClaimBound
+	pvc.Status.Capacity = map[corev1.ResourceName]resource.Quantity{
+		corev1.ResourceStorage: *resource.NewQuantity(request, resource.BinarySI),
```

Comment:
Don't you need a test case where the condition `current_capacity > requested_capacity + increase_step` holds?
Though your code changes looks good, I would be glad if you could add an explicit test for the original problem. 

I guess you can add a parameter to the following struct to express the value of `pvc.Status.Capacity` and pass it to createPVC. (In most cases, the new parmeter would be set to the same value with `pvcSizeGi ` variable.)
https://github.com/topolvm/pvc-autoresizer/blob/1c1ba80c35130ece49018f36bbdfefbf82cd7ad0/runners/pvc_autoresizer_test.go#L171-L180

Because I'm relatevely new to this project, I would also like to hear your opinioin. (Is it overkill to add tests for each bug fix?)

Reply:
I added test cases. Would you check these again?

Reply:
LGTM. Thank you for your effort.